styles/components/alert.scss

```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
+	position: relative;
+	color: $brand-white;
 
+	.dismiss-btn {
+		float: right;
+		background: transparent;
+		color: $brand-white;
+    border: 1px solid $brand-white;
+    width: 20px;
+    height: 20px;
+    text-align: center;
+    line-height: 16px;
+    cursor: pointer;
+    border-radius: $border-radius-rounded;
+   	-webkit-transition: background,color 0.3s linear;
```

Comment:
Leave out vendor prefixes. We'll use [autoprefixer](https://github.com/postcss/autoprefixer) to do this for us (automation) on release.